application/cms/models.py

```diff
@@ -596,54 +596,28 @@ def dimension_classification(self):
     # the highest number of ethnicities if both exist.
     def update_dimension_classification_from_chart_or_table(self):
 
-        classification_id = None
-        includes_parents = None
-        includes_all = None
-        includes_unknown = None
+        chart_or_table = None
 
         if self.dimension_chart and (self.dimension_table is None):
-
-            # Copy settings from chart
-            classification_id = self.dimension_chart.classification_id
-            includes_parents = self.dimension_chart.includes_parents
-            includes_all = self.dimension_chart.includes_all
-            includes_unknown = self.dimension_chart.includes_unknown
-
+            chart_or_table = self.dimension_chart
         elif self.dimension_table and (self.dimension_chart is None):
-
-            # Copy settings from table
-            classification_id = self.dimension_table.classification_id
-            includes_parents = self.dimension_table.includes_parents
-            includes_all = self.dimension_table.includes_all
-            includes_unknown = self.dimension_table.includes_unknown
-
+            chart_or_table = self.dimension_table
         elif self.dimension_table and self.dimension_chart:
-
             if (
-                self.dimension_table.classification.ethnicities_count
-                > self.dimension_chart.classification.ethnicities_count
+                self.dimension_chart.classification.ethnicities_count
+                > self.dimension_table.classification.ethnicities_count
```

Comment:
@TheDoubleK did you deliberately reverse these? I think it makes the logic wrong? (We want to select the chart or table with the highest number of ethnicities...)

Reply:
Umm... well done in spotting the deliberate mistake, 10 points :tada:

Reply:
More to the point... why are the tests passing :man_facepalming: